### ISSUE TITLE
Pact wizard

### ISSUE THREAD

**Issue by K1hana** [kreaddy/MysticalMayhem#6]
Pact wizard is not working with archetype stacking.Pact Wizard/Exploiter Wizard will get a strange ability at 10th level，and spell bar will unable to open
![BUG](https://user-images.githubusercontent.com/116928500/198791726-8974a7bf-7abe-440f-8af4-ab8b8175bd27.png)

**Comment by kreaddy** [comment]
Works normally during testing.

![test](https://user-images.githubusercontent.com/95177331/198830391-9466b16d-33ed-4b41-9908-b9b4fc9e27dd.jpg)

**Comment by K1hana** [comment]
Emmmm.It happens after you prepare your spells.

**Comment by kreaddy** [comment]
Nope, it doesn't happen to me. I don't know what's happening to you but if you want me to do anything about it you'll have to provide me with ways to reproduce it.

**Comment by K1hana** [comment]
https://user-images.githubusercontent.com/116928500/198840241-8aacbc74-a915-4306-98ba-e086b4490039.mp4

![OFK71 NY0SH 8XG50P794L1](https://user-images.githubusercontent.com/116928500/198840258-ef97e3ee-22c2-41ae-8ce3-cbe07f8029ed.png) These are my mods

**Comment by K1hana** [comment]
I discover its question ，it only happen when I choose Elements patron！

**Comment by kreaddy** [comment]
> I discover its question ，it only happen when I choose Elements patron！

That's a pretty important detail! Will investigate.

**Comment by kreaddy** [comment]
Okay, I confirmed it. It's indeed specific to the Elements patron.

**Comment by kreaddy** [comment]
All right. The problem had nothing to do with archetype stacking and would have happened on a normal pact wizard just the same. For some reason, the way I used to detect spells with variants would incorrectly flag Flame Strike as one, making the game try to add non-existent variants to be cast spontaneously.

I fixed it but since I won't push a release just for this, here's a hotfix in the meantime. Just paste the files in the mod folder.
[hotfix.zip](https://github.com/kreaddy/MysticalMayhem/files/9894148/hotfix.zip)
